runtime/interpreter/encode.go

```diff
@@ -681,13 +681,23 @@ func (v UFix64Value) Encode(e *atree.Encoder) error {
 	return e.CBOR.EncodeUint64(uint64(v))
 }
 
-// Encode encodes SomeStorable as
+var _ atree.ContainerStorable = &SomeStorable{}
+
+func (s SomeStorable) Encode(e *atree.Encoder) error {
+	nonSomeStorable, nestedLevels := s.nonSomeStorable()
+	if nestedLevels == 1 {
+		return s.encode(e)
+	}
```

Comment:
can this also be handled as part of encodeMultipleNestedLevels (instead of returning error there) ?

Reply:
Sorry for delay in reply, I missed this comment.  :pray: 

Maybe not in this case because encoding format is differently for `nestedLevel == 1` (using `encode()`)  vs `nestedLevel > 1` (using `encodeMultipleNestedLevels()`):
- if nested level == 1, only innermost unwrapped storable is encoded
- if nested level > 1, then 2-element array is encoded: first element is nested levels, second element is innermost unwrapped storable.